README.md

```diff
@@ -4,13 +4,70 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
-`gem 'webmock-fixtures`
+`gem "webmock-fixtures"`
+
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "rspec"
+require "webmock/fixtures"
+require "webmock/rspec"
+
+# Register our fixture for www.example.org
+WebMock::Fixtures::Manager.register_fixture(
+  :get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+RSpec.describe do
+  describe "a web request to www.example.org" do
+    let!(:manager) { WebMock::Fixtures::Manager.run([:get_example]) }
```

Comment:
Ah, nope. The `webmock/rspec` takes care of that as you mentioned in another PR. Maybe highlight the significance of that line (so people will know to teardown otherwise).

https://github.com/bblimke/webmock/blob/b18b029ab9e4f9556cae590c1cb72beee54ae853/lib/webmock/rspec.rb#L23-L31